processing/src/main/java/org/apache/carbondata/processing/loading/partition/impl/RawRowComparator.java

```diff
@@ -34,12 +36,34 @@
   private int[] sortColumnIndices;
   private boolean[] isSortColumnNoDict;
   private DataType[] noDicDataTypes;
+  private Map<DataType, SerializableComparator> comparator_map;
 
   public RawRowComparator(int[] sortColumnIndices, boolean[] isSortColumnNoDict,
       DataType[] noDicDataTypes) {
     this.sortColumnIndices = sortColumnIndices;
     this.isSortColumnNoDict = isSortColumnNoDict;
     this.noDicDataTypes = noDicDataTypes;
+    comparator_map = new HashMap<>();
```

Comment:
Now this code is not required..Please remove

Reply:
Removed